lale/lib/sklearn/ordinal_encoder.py

```diff
@@ -331,5 +331,36 @@ def inverse_transform(self, X):
             set_as_available=True,
         ),
     )
+    OrdinalEncoder = OrdinalEncoder.customize_schema(
+        constraint={
+            "description": "unknown_value should be an integer or np.nan when handle_unknown is 'use_encoded_value'. s",
+            "anyOf": [
```

Comment:
The description says that `np.nan` is also okay. So the `"anyOf"` should probably allow for a third alternative:
```python
{
    "type": "object",
    "properties": {"unknown_value": {"enum": [np.nan]}},
},
```